app/models/ems_refresh/save_inventory.rb

```diff
@@ -20,6 +20,7 @@ def save_ems_inventory(ems, hashes, target = nil)
     when ManageIQ::Providers::StorageManager                then save_ems_storage_inventory(ems, hashes, target)
     when ManageIQ::Providers::MiddlewareManager             then save_ems_middleware_inventory(ems, hashes, target)
     when ManageIQ::Providers::DatawarehouseManager          then save_ems_datawarehouse_inventory(ems, hashes, target)
+    when ManageIQ::Providers::PhysicalInfraManager          then save_ems_physical_infra_inventory(ems, hashes, target)
```

Comment:
This line is already in master https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_refresh/save_inventory.rb#L14

We have alias for `EmsCloud`, `EmsInfra`, and `EmsPhysicalInfra`